demo/index.html

```diff
@@ -29,4 +99,38 @@
 
   <!-- Demo -->
     <script src="annotator-content-analysis_demo.js"></script>
+
+  <!-- temp -->
+    <script>
+      // this is hacky and belongs somewhere else
+      var currentTopic = "1";
+      function collapse(elem) {
+        $(elem).parent().children().toggle();
+        $(elem).toggle()
+               .toggleClass('variable-list__collapsible--closed');
+      }
+      function activateTopic(topic) {
+        $('.topic-picker__nav li').removeClass('active');
+        $(`.topic-picker__nav li[data-topic="${topic}"]`).addClass('active');
+        $('.text-wrapper__text').attr('data-topic', topic);
+
+        $('.topic-wrapper').hide();
+        $(`.topic-wrapper[data-topic="${topic}"]`).show();
+      }
+
+      $(document).ready(_ => {
+        $('.topic-picker__pin-button').click(function() {
+          $('.topic-picker').toggleClass('topic-picker--open');
+          $(this).toggleClass('fa-inverse');
+        });
+        $('.variable-list .variable-list__collapsible').click(function() {
+          collapse(this);
+        }).click();
```

Comment:
This handles the collapsing of the variable list, so that variable A can have subvariables 1 and 2 hidden or shown. It's called so that things are hidden by default.